### PR TITLE
tests: Add LDAP URI to ldappasswd explicitelly

### PR DESCRIPTION
**Pull request by davidkupka**
https://fedorahosted.org/freeipa/ticket/6622

This PR replaces PR #404 which is orphaned and I'm no longer able to push into it.

### CODE REVIEW

File: ipatests/pytest_plugins/integration/tasks.py [hunk 1]
```diff
@@ -1314,7 +1314,8 @@ def ldappasswd_user_change(user, oldpw, newpw, master):
     basedn = master.domain.basedn
 
     userdn = "uid={},{},{}".format(user, container_user, basedn)
+    master_ldap_uri = "ldap://{}:389".format(master.external_hostname)
```
Comment (rcritten) [inline ipatests/pytest_plugins/integration/tasks.py]: nit: is the 389 really necessary? It is the default when no port is used for an ldap uri.

Comment (davidkupka) [inline ipatests/pytest_plugins/integration/tasks.py]: You're right, it's useless. Thanks.

**Comment by rcritten** [comment]
I rebased the patch and updated the commit message.

**Comment by abbra** [comment]
LGTM. Tests pass too.

**Comment by tomaskrizek** [comment]
master:

* 53abf0105b3dd6d5e2b258375c03e6ecc23bc162 tests: Add LDAP URI to ldappasswd explicitly

**Pull request closed without merging**